go.mod

```diff
@@ -0,0 +1,76 @@
+module sigs.k8s.io/aws-fsx-openzfs-csi-driver
+
+go 1.20
```

Comment:
did you test mounting after updating this and building a new image

Reply:
Yep